buildtest/cli/compilers.py

```diff
@@ -211,18 +215,24 @@ def find_compilers(self):
         module_dict = {}
         console.print(f"MODULEPATH: {os.getenv('MODULEPATH')}")
 
+        print("self.moduletool: ", self.moduletool)
```

Comment:
I think this print statement was for debugging purposes right? can you remove it.